lib/common/utils.c

```diff
@@ -1199,3 +1199,10 @@ pcmk_hostname()
 
     return (uname(&hostinfo) < 0)? NULL : strdup(hostinfo.nodename);
 }
+
+char *
+date_string(time_t epoch_time) {
+    static char empty_str[] = "";
+    char *s = asctime(localtime(&epoch_time));
+    return s == NULL ? empty_str : crm_strip_trailing_newline(s);
+}
```

Comment:
I don't like adding another function like this and if I could accomplish the same thing in another way (perhaps even by reusing code we already have), I would love to.